annotate.py

```diff
@@ -225,10 +226,45 @@ def annotate_dir(img_dir, initials, size1=1, size2=1) :
         textName = os.path.join(imageName+".txt")
         maskName = os.path.join(imageName+".png")
 
-        print("------- Moving ",textName, "to ", text_save_path)
-        shutil.move(textName,text_save_path)
-        print("------- Moving ",maskName, "to ", mask_save_path)
-        shutil.move(maskName,mask_save_path)
+
+        saveText=True
+        if(os.path.exists(os.path.join(text_save_path,textName))):
+            print("File", textName, "already exists in", text_save_path+". Would you like to overwrite it?")            
+            overwriteText = input("(y/n)\n-->")
+            if(overwriteText == "y" or overwriteText == "Y"):
+                saveText = True
+                os.remove(os.path.join(text_save_path,textName))
+            else:
+                saveText = False
+
+        if(saveText == True):
+            print("Moving",textName, "to", text_save_path)        
+            shutil.move(textName,text_save_path)
+        else:
+            os.remove(textName)
+            print("Removed new", textName)
+
+
+        print("\n")
+
+
+        saveMask=True
+        if(os.path.exists(os.path.join(mask_save_path,maskName))):
+            print("File", maskName, "already exists in", mask_save_path+". Would you like to overwrite it?")
+            overwriteMask = input("(y/n)\n-->")
+            if(overwriteMask == "y" or overwriteMask == "Y"):
```

Comment:
for case-insensitivity, use the `.lower()` function ([ref](https://stackoverflow.com/questions/319426/how-do-i-do-a-case-insensitive-string-comparison)).

line 234 above too

Reply:
Change added.